src/Kafka/CreateKafkaInstance/components/ModalAlerts.tsx

```diff
@@ -53,6 +61,31 @@ export const ModalAlerts: VoidFunctionComponent<ModalAlertsProps> = ({
                 {t("modal_alerts.system_unavailable_message")}
               </Alert>
             );
+          case isTesting && instanceAvailability === "trial":
+            return (
+              <Alert
+                id="mk-create-instance-quota-alert"
```

Comment:
I think I removed this id in the other alerts. Could you check if this is the case?